coalib/bearlib/aspects/Smell.py

```diff
@@ -101,19 +95,80 @@ def func(a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t, z):
         parameters (maybe by creating more functions or using libraries).
         """
 
+
+@MethodSmell.subaspect
+class MethodLength:
+    """Illustrate code smells related to methods' length.
+
+    `MethodLength` refers to the number of lines of code in a function or
+    method definition. Depending on the value of `method_length_count_comment`,
+    comments are considered.
+    """
+    class docs:
+        example = """
+        def is_positive(num):
+            if num >= 0:
+                return True
+            else:
+                return False
+        # This function can be defined in one line as follows:
```

Comment:
per https://github.com/coala/coala/pull/4398#discussion_r128909085 ; please add a blank line above this.